### ISSUE TITLE
Error writing header: connection refused / Could not write header for output since 1.5.0

### ISSUE THREAD

**Issue by hamsando** [mrlt8/docker-wyze-bridge#417]
Running happily on 1.4.5, but upgrading to anything newer results in immediate failures. Nothing else changed in my setup. The log below is from 1.5.0, but I've also tried all the way up to 1.5.3 with the same results:
```🚀 STARTING DOCKER-WYZE-BRIDGE v1.5.0

2022/06/13 21:01:10 [WyzeBridge] 🔍 Could not find local cache for 'user'
2022/06/13 21:01:10 [WyzeBridge] 🔍 Could not find local cache for 'auth'
2022/06/13 21:01:10 [WyzeBridge] ☁️ Fetching 'auth' from the Wyze API...
2022/06/13 21:01:10 [WyzeBridge] 💾 Saving 'auth' to local cache...
2022/06/13 21:01:10 [WyzeBridge] ☁️ Fetching 'user' from the Wyze API...
2022/06/13 21:01:11 [WyzeBridge] 💾 Saving 'user' to local cache...
2022/06/13 21:01:11 [WyzeBridge] 🔍 Could not find local cache for 'cameras'
2022/06/13 21:01:11 [WyzeBridge] ☁️ Fetching 'cameras' from the Wyze API...
2022/06/13 21:01:11 [WyzeBridge] 💾 Saving 'cameras' to local cache...

🪄 BLACKLIST MODE ON

🎬 STARTING 2 OF 3 CAMERAS
2022/06/13 21:01:11 [WyzeBridge] Starting rtsp-simple-server v0.19.0
2022/06/13 21:01:11 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Overhead on 192.168.0.97 (1/3)
2022/06/13 21:01:11 [WyzeBridge] 🎉 Connecting to WyzeCam V2 - Garage on 192.168.0.93 (1/3)
ERR: The minimum number of HLS segments is 3
2022/06/13 21:01:13 [Garage] 📡 Getting 120kb/s HD stream (15fps) via LAN mode (WiFi: 82%) FW: 4.9.8.1002 🔒 (DTLS) (2/3)
[tcp @ 0x564f81223440] Connection to tcp://0.0.0.0:8554?timeout=0 failed: Connection refused
[tee @ 0x564f81220500] Slave '[select=v:f=rtsp:rtsp_transport=tcp]rtsp://0.0.0.0:8554/garage': error writing header: Connection refused
[tee @ 0x564f81220500] All tee outputs failed.
Could not write header for output file #0 (incorrect codec parameters ?): Connection refused
Error initializing output stream 0:0 -- 

2022/06/13 21:01:14 [Overhead] 📡 Getting 120kb/s HD stream (20fps) via LAN mode (WiFi: 92%) FW: 4.61.0.3 (2/3)
2022/06/13 21:01:14 [Overhead] WARNING: Skipping smaller frame at start of stream (frame_size=1)
[tcp @ 0x55c739510bc0] Connection to tcp://0.0.0.0:8554?timeout=0 failed: Connection refused
[tee @ 0x55c73950f480] Slave '[select=v:f=rtsp:rtsp_transport=tcp]rtsp://0.0.0.0:8554/overhead': error writing header: Connection refused
[tee @ 0x55c73950f480] All tee outputs failed.
Could not write header for output file #0 (incorrect codec parameters ?): Connection refused
Error initializing output stream 0:0 -- 

2022/06/13 21:01:14 [Garage] [Errno 32] Broken pipe
2022/06/13 21:01:14 [WyzeBridge] 🎉 Connecting to WyzeCam V2 - Garage on 192.168.0.93 (1/3)
2022/06/13 21:01:14 [Overhead] [Errno 32] Broken pipe
2022/06/13 21:01:15 [WyzeBridge] 🎉 Connecting to WyzeCam V3 - Overhead on 192.168.0.97 (1/3)
```

at this point in the log, it just start tries again and fails in the same way, over and over again.

I think this seems similar to Issue #415.

**Comment by mrlt8** [comment]
Looks like rtsp-simple-server isn't starting up because of `The minimum number of HLS segments is 3`...

Could you try disabling HLS `RTSP_HLSDIAABLE=True` to see if the issue is coming from there?

**Comment by hamsando** [comment]
In going to add the ENV you suggested (which I assume was supposed to be "RTSP_HLSDISABLE"?), I noticed I had an existing ENV "RTSP_HLSSEGMENTCOUNT" set to 1!

I changed that to 3 and restarted with the latest (1.5.3) and it worked. I don't recall setting that to 1 in the past (or why I would have - although I suppose it was an attempt to reduce latency for some reason that I have long since forgotten about). I guess rtsp-simple-server made some changes in a recent release to **require** a minimum value of 3.

Thanks for the suggested guidance that lead me to the resolution.
